### PR TITLE
Serializer upd8

### PR DESCRIPTION
**Pull request by makosblade**
This PR:
- Add an OrganizationStreamSerializer that serializes everything independently instead of using jackson
- Add a serializer for GrantType
- Add an AbstractEnumSerializer that uses ORDINAL (so don't change the ORDer of enum entries using this)
- Add a Grant Serializer
Depends on ab8a658b388e554ca232df0cffc1e475f57f467e in Rhizome for some more serializers

### CODE REVIEW

File: src/main/kotlin/com/openlattice/hazelcast/serializers/AbstractEnumSerializer.kt [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.openlattice.hazelcast.serializers
+
+import com.google.common.cache.CacheBuilder
+import com.google.common.cache.CacheLoader
+import com.google.common.cache.LoadingCache
+import com.hazelcast.nio.ObjectDataInput
+import com.hazelcast.nio.ObjectDataOutput
+import com.kryptnostic.rhizome.pods.hazelcast.SelfRegisteringStreamSerializer
+
+abstract class AbstractEnumSerializer<T : Enum<T>> : SelfRegisteringStreamSerializer<Enum<T>> {
```
Comment (katherinebernstein) [inline src/main/kotlin/com/openlattice/hazelcast/serializers/AbstractEnumSerializer.kt]: :D

**Comment by makosblade** [comment]
> it would probably be good to have stream serializer tests for these

Huh, didn't we used to have `TestableSelfRegisteringStreamSerializer` that would atuo-test 🤔

**Comment by katherinebernstein** [comment]
> TestableSelfRegisteringStreamSerializer

maaaybe? We have `AbstractStreamSerializerTest` which can be extended very easily, at least

**Comment by makosblade** [comment]
I added a `TestableSelfRegisteringStreamSerializer` tester and guess what, all of this is broken 😆

**Comment by geekbeast** [comment]
Only mapstores have auto-test.

Also, we had an organization stream serializer, I deleted it when I made organization object more complex, because of the deep map nesting. Glad to have it back!

**Pull request merged**